.github/workflows/checks.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       - name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.11.6'
+          python-version: '3.9'
       - name: Install dependencies
         run: pip3 install -r requirements.txt
       - name: Run unit tests
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 28 to 31]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->This should be baked into Ruff itself<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/pocketsizefund/pocketsizefund/153?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->